src/cfg.rs

```diff
@@ -25,6 +26,7 @@ impl Default for Config {
             fnames: connectors::fname::Config::default(),
             onchain_events: connectors::onchain_events::Config::default(),
             consensus: consensus::consensus::Config::default(),
+            rpc_address: "0.0.0.0:2283".to_string(),
```

Comment:
We should pick something like 3383. Also, update setup.rs to generate ports that don't collide for the different nodes.